Task 2/Task 2.1/Task 2.1.2/Figures/RoundShapes/Circle.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Task_2._1._2.Figures.RoundShapes
+{
+    public class Circle : Figure
+    {
+        public Point Centre { get; }
+        public int Radius { get; }
+        public Circle(Point p, int r) : base("Окружность")
```

Comment:
Use empty lines to segregate different groups of methods (ctors from properties, multiline members from each other, etc.)